modules/subscriptions/subscriptions/subscriptions.test.tsx

```diff
@@ -0,0 +1,197 @@
+import "@testing-library/jest-dom";
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { CoreComponent } from "@/core/module/module.types";
+import "./index";
+import "@/services/auth/auth-mock.service";
+import "@/services/subscriptions/subscriptions-mock.service";
+
+import React, { act } from "react";
+import { getComponent } from "@/resources/tests/test.utils";
+import Service from "@/core/module/service.factory";
+import { jest } from "@jest/globals";
+
+jest.mock("@/core/module/module.manager", () => ({
+  ModuleManager: {
+    get: jest.fn().mockReturnValue({
+      components: {
+        services: jest.fn(),
+      },
+    }),
+  },
+}));
+
+describe("===== PAYMENTS =====", () => {
+  describe("Subscriptions Page", () => {
+    let Subscriptions: CoreComponent;
+
+    beforeAll(() => {
+      Subscriptions = getComponent("app", "subscriptions");
+    });
+
+    beforeEach(() => {
+      delete window.location;
+      window.location = { href: "" } as Location;
```

Comment:
Tell me more about it
I think in `setup-jest` there is already some mocks for it.